pkg/framework/test/integration/integration_test.go

```diff
@@ -27,11 +27,11 @@ var _ = Describe("Integration", func() {
 
 		Eventually(func() bool {
 			return isSomethingListeningOnPort(2379)
-		}, 5*time.Second).Should(BeTrue(), "Expected Etcd to listen on 2379")
+		}, 25*time.Second).Should(BeTrue(), "Expected Etcd to listen on 2379")
```

Comment:
I foresee :snowflake:  :-)

Reply:
Yeah ... we need to figure out how to handle that properly -- we dropped [a story](https://www.pivotaltracker.com/story/show/153245508) in our backlog for that.